eclair-core/src/main/resources/reference.conf

```
@@ -99,6 +99,7 @@ eclair {
     min-public-funding-satoshis = 100000
     min-private-funding-satoshis = 100000
     max-funding-satoshis = 16777215 // to open channels larger than 16777215 you must enable the large_channel_support feature in 'eclair.features'
+    require-confirmed-inputs-for-funding = false // if true, only confirmed inputs will be accepted for channel funding
```

Nits:
- we should at some point group under a `funding` section
- 
```suggestion
    require-confirmed-inputs-for-dual-funding = false // if true, only confirmed inputs will be accepted for dual-funding
```

---

Good idea to create a `funding` section. Since we already have other fields to move to that new function section, I'll do that in a follow-up PR.

---

Renamed in https://github.com/ACINQ/eclair/pull/2461/commits/68bfcae48a269d6ded0c5966a06167e4984cdc31